inference/skos_patch.ttl

```diff
@@ -0,0 +1,134 @@
+#################################################################
```

Comment:
Have you tried adding an import to SKOS in the header instead? Having a patch like this is what's professionally called "ugly" :)

Reply:
SKOS import is done automatically, in fact this file is used to unmerge unneccesary properties. Please read our final report to find more details on this.

Reply:
Ah! Okay. Then please document what this file is for in the repo. Something like `inference/README.md`.